main/guides/getting-started/README.md

```diff
@@ -1,163 +1,199 @@
-# Installing the Agoric SDK
+# Starting a Basic Dapp
 
-To write JavaScript smart contracts using the Agoric Zoe framework, first install the Agoric Software
-Development Kit (SDK).
+Welcome! We'll start with a basic _dapp_; that is: a JavaScript contract and UI.
 
-After installing the Agoric SDK, you can proceed to [starting a project](./start-a-project.md).
+But first, note that help is not far away:
 
 ## Getting Support
 
 - Chat with peers on the [Agoric Discord](https://agoric.com/discord)
 - Join weekly [Office Hours](https://github.com/Agoric/agoric-sdk/wiki/Office-Hours)
 - Search and post [Q & A](https://github.com/Agoric/agoric-sdk/discussions/categories/q-a) in [agoric-sdk discussions](https://github.com/Agoric/agoric-sdk/discussions)
 
-## Quick Start
+## Platform Requirements
 
-If you're on a [supported platform](#platform-linux-shell-or-equivalent) (MacOS, Linux, or WSL) and you're familar with JavaScript development tools such as `node`, `yarn`, and `git`:
+We support any long-term support (LTS) versions of Node.js.
+Download and install from [Node.js](https://nodejs.org/) if you don't
+already have `node`. Check to make sure:
 
 ```shell
-go version # Version 1.20.3 or higher
 node --version # LTS version such as 18.16.0
-npm install --global yarn # Install yarn for package management
-git clone --branch community-dev https://github.com/Agoric/agoric-sdk # Clone the "community-dev" branch
-cd agoric-sdk
-yarn install # Asks yarn to install all the dependant node packages
-yarn build # Builds the agoric-sdk packages
-(cd packages/cosmic-swingset && make) # Builds the cosmic-swingset package
-yarn link-cli ~/bin/agoric # Creates an executable script
-agoric --version # Prints the version number of the SDK
 ```
 
-Now you are ready proceed to [starting a project](./start-a-project.md).
+We use the `yarn` package manager. [Installing with corepack](https://yarnpkg.com/corepack) is what the `yarn` maintainers recommend:
 
-_If you get "command not found", see [troubleshooting below](#install-agoric-cli)._
-
-::: tip Watch: Prepare Your Agoric Environment (November 2020)
-This presentation is a good overview of the Agoric SDK setup process,
-though a few details are out of date:
-
-- node version: 12.x is too old; use the LTS version 18.16.0 or a later LTS version
-- skip `git checkout hackathon-2020-11`; use the `community-dev` branch
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/w0By22jYhJA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-**Note:** Video omits adding the Agoric SDK to your PATH.
-:::
-
-## Platform: Linux Shell or Equivalent
+```sh
+corepack enable
+yarn --version # 1.x
+```
```

Comment:
I agree it's best to recommend yarn here, but is it also worth highlighting that other package managers work? 

I had success with `pnpm` and `npm` like mentioned [here](https://github.com/Agoric/agoric-sdk/pull/8530/files#diff-c4ec3b24fc816864e385653a5d06ca6cdc09281ed10bd5c28e64ae781c75374bR3-R20).


I was also curious to give yarn 4.x a try, and had luck with this:

```bash
# upgrade from yarn classic to yarn 4.x
yarn set version berry 

# disable PnP modules
yarn config set nodeLinker node-modules
```

Reply:
How about a separate issue?

We would want to set up ci to ensure that other package managers continue to work.